### PR TITLE
Adds issue template

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

File: templates/github/.github/ISSUE_TEMPLATE/task.md.j2 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: 🗒️ Task
+about: Documentation changes, CI/CD changes, work items such as "investigate X"
+title: ''
+labels: Task, Triage-Needed
```
Comment (lubosmj) [inline templates/github/.github/ISSUE_TEMPLATE/task.md.j2:5]: A task does not need to be flagged as `Triage-Needed`.

**Review by lubosmj — CHANGES_REQUESTED** [review]
Would you mind also adding a template for stories?

File: templates/github/.github/ISSUE_TEMPLATE/feature_request.md.j2 [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: ✨ Feature request
+about: Suggest an idea for this project
+title: ''
+labels: Feature, Triage-Needed
```
Comment (lubosmj) [inline templates/github/.github/ISSUE_TEMPLATE/feature_request.md.j2:5]: A feature request should not be marked as `Triage-Needed` as well. Am I correct?

File: templates/github/.github/ISSUE_TEMPLATE/task.md.j2 (hunk 1 above)
Comment (dralley) [reply templates/github/.github/ISSUE_TEMPLATE/task.md.j2:5]: Per the old process, yes.

But I don't think there's a great reason to exclude stories or tasks from the triage process personally.  They all ought to have some level of review.

**Comment by fao89** [comment]
> Would you mind also adding a template for stories?

feature == stories

File: templates/github/.github/ISSUE_TEMPLATE/task.md.j2 (hunk 1 above)
Comment (fao89) [reply templates/github/.github/ISSUE_TEMPLATE/task.md.j2:5]: on the previous process, 
the default was an issue and everything went to triage, and users rarely changed that.

**Review by dralley — APPROVED** [review]

**Review by lubosmj — APPROVED** [review]

**Pull request merged**